Description: Don't install OpenSSL docs
 /usr/bin/pod2man (or rather Pod::Simple) from perl 5.18.1-4 is unhappy with openssl-1.0.1e.
 Let's avoid installing docs by making install_sw instead of install.
 .
 shadow (1.8.0-1) unstable; urgency=low
 .
   * Initial release
Author: Shadow Support <shadow-support@cs.umn.edu>

--- shadow-1.8.0.orig/setup
+++ shadow-1.8.0/setup
@@ -229,7 +229,7 @@ def dependencies(args):
     configure = "./config --prefix={0} shared threads {1} -fPIC".format(args.prefix, ecopt)
 
     log("now attempting to build openssl with '{0}'".format(configure))
-    if automake(configure, "make", "make install") != 0: 
+    if automake(configure, "make", "make install_sw") != 0: 
         log("ERROR!: problems building {0}".format(openssl_tarball))
         return -1
         
